src/plugins/ui_core_views/custom_colors.ts

```diff
@@ -90,6 +93,17 @@ export class CustomColorsPlugin extends UIPlugin<CustomColorState> {
       case "SET_ZONE_BORDERS":
       case "SET_FORMATTING":
         this.history.update("shouldUpdateColors", true);
+        break;
+      case "START":
+      case "CREATE_TABLE":
+      case "UPDATE_TABLE":
+      case "REMOVE_TABLE":
+        this.history.update("tableColors", this.getTableColors());
```

Comment:
why have a different strategy for table colors than for other colors ?
table colors: have `this.tableColors` and update it directly
other colors: set `this.shouldUpdateColors` to true and compute the colors

Reply:
Because even if it isn't expensive, it seemed a bit dumb to recompute the table colors at each UPDATE_CELL.

That being said, we will need to recompute the table colors at each UPDATE_CELL after dynamic tables are introduced, so I can probably just remove the optimization.